lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # exercise car model
+  class Car
+    attr_reader :year, :color, :model, :current_speed
+    def self.default_car
+      Car.new {}
+    end
+
+    def initialize(args)
+      raise ArgumentError if args.class != Hash
```

Comment:
Use `is_a?`, `instance_of?` instead of class comparison. Please, google the differences between them.